### PR TITLE
Feature/graphql ui

### PR DESCRIPTION
**Pull request by peng-yongsheng**
Please answer these questions before submitting pull request

- Why submit this pull request?
- [ ] Bug fix
- [x] New feature provided
- [ ] Improve performance

### New feature or improvement
- Describe the details and related test reports.

Implement the method named getAllApplication in application layer graphql protocol.

### CODE REVIEW

**Comment by hanahmily** [comment]
I will adapt the UI for the next two days. Once this is done, the pr could be merged.

**Comment by hanahmily** [comment]
@peng-yongsheng @wu-sheng 
Before merge, There are still some errors to fix .
    1. Support 'variables', using `Gson` to parse the 'variables' node of post payload to a `java.util.Map` object.
    1. Handle exception carefully. `GraphQLHandler#doPost` method CAN NOT throw exception.

There are some issues in '.graphqls' files. I will create a pr to fix them.

**Comment by peng-yongsheng** [comment]
> 1. Support 'variables', using Gson to parse the 'variables' node of post payload to a java.util.Map object.

I provide getAllApplication and getApplicationTopology methods only, is there any scenes that need to use 'variables'? I will do it when that other methods need.

**Comment by hanahmily** [comment]
Sorry for the late reply. There is one issue for `getApplicationTopology` query.

## `getAllApplication` query can return some data
### query 
```graphql
query ApplicationOption($duration: Duration!) {
   getAllApplication(duration: $duration) {
    key: id
    label: name
  }
}
```
### variables
```json
{"duration":{"start":"2018-02-02 1538","end":"2018-02-02 1638","step":"MINUTE"}}
```

### data

```json
{
  "data": {
    "getAllApplication": [
      {
        "key": "-2",
        "label": "172.25.0.4:20880"
      },
      {
        "key": "1",
        "label": "User"
      },
      {
        "key": "3",
        "label": "localhost:27017"
      }
    ]
  }
}
```

## But `getApplicationTopology` query CAN NOT return any data
### query 
```graphql
query Application($applicationId: ID!, $duration: Duration!) {
  getApplicationTopology(applicationId: $applicationId, duration: $duration) {
    nodes {
      id
      name
      type
      ... on ApplicationNode {
        sla
        callsPerSec
        responseTimePerSec
        apdex
        isAlarm
        numOfServer
        numOfServerAlarm
        numOfServiceAlarm
      }
    }
    calls {
      source
      target
      isAlert
      callType
      callsPerSec
      responseTimePerSec
    }
  }
}

```
### variables
```json
{"duration":{"start":"2018-02-02 1538","end":"2018-02-02 1638","step":"MINUTE"},"applicationId":"3"}
```

### data

```json
{
  "data": {
    "getApplicationTopology": {
      "nodes": [],
      "calls": []
    }
  }
}
```

I also confirm the arguments of `org.apache.skywalking.apm.collector.ui.service.ApplicationTopologyService#getApplicationTopology` are correct.

Looking forward to @peng-yongsheng reply

**Comment by hanahmily** [comment]
> I provide getAllApplication and getApplicationTopology methods only, is there any scenes that need to use 'variables'? I will do it when that other methods need.

I use variables for all arguments. So I send a commit to enhance `GraphQLHandler` to parse variables

File: apm-collector/apm-collector-agent/agent-jetty/agent-jetty-provider/src/test/resources/json/dubbox-provider.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
               "pii": 2, //上级的实例编号
```
Comment (wu-sheng) [inline apm-collector/apm-collector-agent/agent-jetty/agent-jetty-provider/src/test/resources/json/dubbox-provider.json]: Please don't Chinese characters in the source codes.

**Comment by wu-sheng** [comment]
@peng-yongsheng @hanahmily I am waiting for the CI, and prepare to merge this pr. Please start a new pr for further features and bug fixes.

**Pull request merged**